manifests/gmond.pp

```diff
@@ -65,4 +65,7 @@
     enable     => true,
     status     => $::ganglia::params::gmond_status_command,
   }
+  if ($operatingsystem == 'RedHat' or $operatingsystem == 'CentOS') and ($operatingsystemmajrelease == '7' or $operatingsystemmajrelease == '6') {
```

Comment:
This line is failing the linter for whitespace.  Top level variables should be fully qualified.  Eg., `$::operatingsystem`.  Inside of doing multiple `$::operatingsystem` checks for EL derived platforms the `$::osfamily` fact can be used.  Eg. `$::osfamily == 'RedHat'`.

However, it would be better to convert `$::ganglia::params::gmond_package_name` to an array for EL 6 & 7 and add the new package to that.  Does that make sense to you?